tests/compilation_cache_test.py

```diff
@@ -587,6 +604,26 @@ def test_jit(self):
       f(1)
       self.assertEqual(count_cache_items(), 0)
 
+  def test_enable_compilation_cache(self):
```

Comment:
nit: should this be after the other test? Maybe it doesn't make a difference, but they're very similar, so I think it's good to put them together.